local/public/wp-content/plugins/hello.php

```diff
@@ -3,3 +3,6 @@
  * Plugin Name: Hello Dolly
  * Description: A plugin used for PHP unit tests
  */
+
+
+

```

Comment:
This is so annoying -- I don't think it really matters but why is there a test that affects this file? Maybe we should take it out?

Reply:
I hate rolling back this change every time, too! 👿 

> Maybe we should take it out?

One hundred percent! 😄

Reply:
I've updated the test so that it puts back the original contents of the file when it's done. It's working for me 😅

Reply:
@delawski The updated test is here https://github.com/xwp/stream/pull/1529/files#diff-b72d8df01c58f9124052f8296360e9aa1a7bab799c60b038171ca3a23b9aa1c8